layers/meta-resin-raspberrypi/recipes-bsp/u-boot/u-boot-common_2018.07.inc

```diff
@@ -0,0 +1,15 @@
+HOMEPAGE = "http://www.denx.de/wiki/U-Boot/WebHome"
```

Comment:
Please specify in the commit log the revision from which you imported this recipe.